### ISSUE TITLE
SConstruct "use_mingw=yes" flag still uses MSVC tooling

### ISSUE THREAD

**Issue by Fr0sk** [godotengine/godot-cpp#337]
I found out today that compiling the `godot-cpp` repository using the following command: `scons platform=windows generate_bindings=yes use_mingw=yes`, the build system would resort to `cl.exe`, and thus not relying on the MinGW `g++.exe`. 

To fix this I added:
```
elif host_platform == 'windows' and env['use_mingw']:
        env = env.Clone(tools=['mingw'])
        env["SPAWN"] = mySpawn
```

below the if statement for the linux or osx host: `elif host_platform == 'linux' or host_platform == 'osx':`.

I wonder if this is the intended behavior or if you would like me to submit a PR for this.

Edit: Forgot to add that I'm trying this on Windows 10 Pro x64 build 1903

**Comment by Fr0sk** [comment]
Well, I'm really sorry, but I was using an old version of the SConstruct file and can confirm that the current version works well with MinGW. Closing this issue because of that.

**Comment by Zylann** [comment]
I guess this can be closed now the fix has been merged?

**Comment by Faless** [comment]
Closing as fixed via #360
